### DISCUSSION TITLE
Webpack error with CSS loader

### DISCUSSION THREAD

**Discussion by erosman** [webpack/webpack#19096] [Q&A] [answered]
I am new in using webpack. I was trying to bundle Monaco Editor.

Ubuntu 24.04

├── css-loader@7.1.2
├── monaco-editor@0.52.2
├── npm@10.7.0
├── style-loader@4.0.0
├── webpack-cli@6.0.1
└── webpack@5.97.1

- Copied the content of  [browser-esm-webpack](https://github.com/microsoft/monaco-editor/tree/main/samples/browser-esm-webpack) to `/home/src` directory
- Tried `npx webpack build`
- Got lots of errors

```
ERROR in ./node_modules/monaco-editor/esm/vs/base/browser/ui/actionbar/actionbar.css 6:0
Module parse failed: Unexpected token (6:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|  *--------------------------------------------------------------------------------------------*/
| 
> .monaco-action-bar {
| 	white-space: nowrap;
| 	height: 100%;
 @ ./node_modules/monaco-editor/esm/vs/base/browser/ui/actionbar/actionViewItems.js 15:0-25
 @ ./node_modules/monaco-editor/esm/vs/editor/contrib/contextmenu/browser/contextmenu.js
 @ ./node_modules/monaco-editor/esm/vs/editor/editor.all.js 18:0-54
 @ ./node_modules/monaco-editor/esm/vs/editor/edcore.main.js 5:0-25
 @ ./node_modules/monaco-editor/esm/vs/editor/editor.main.js 7:0-30 7:0-30
 @ ./src/index.js 1:0-40 24:0-20

ERROR in ./node_modules/monaco-editor/esm/vs/base/browser/ui/aria/aria.css 6:0
Module parse failed: Unexpected token (6:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
```

[webpack.config.js](https://github.com/microsoft/monaco-editor/blob/main/samples/browser-esm-webpack/webpack.config.js)
```
        test: /\.css$/,
        use: ['style-loader', 'css-loader']
```

For example, above seems to refer to `import './actionbar.css';` in `actionViewItems.js`
Any ideas?

**Comment by alexander-akait** [comment]
Sorry for delay, do you still have the problem?

**Comment by erosman** [comment, marked as answer]
After some testing, it seems that the Webpack doesnt locate `webpack.config.js`

ATM, it is placed in `/home/src/webpack.config.js` as per [browser-esm-webpack](https://github.com/microsoft/monaco-editor/tree/main/samples/browser-esm-webpack)

Moving it to `/home/webpack.config.js` results in `require not defined`
I managed to get webpack to output (with some errors) with:
```
npx webpack build --config src/webpack.config.js
```
